federatedscope/core/fed_runner.py

```diff
@@ -41,6 +44,9 @@ def __init__(self,
         self.gpu_manager = GPUManager(gpu_available=self.cfg.use_gpu,
                                       specified_device=self.cfg.device)
 
+        # get device information
+        self.device_info = get_device_info(config.federate.device_info_file)
```

Comment:
it looks confusing. `device` always makes me think about gpu and cpu.

Reply:
Change `device_info` to `resource_info`